Provider/ImageProvider.php

```diff
@@ -174,6 +174,10 @@ public function generatePublicUrl(MediaInterface $media, $format)
      */
     public function generatePrivateUrl(MediaInterface $media, $format)
     {
+        if ($format == 'reference') {
```

Comment:
I'm fine with whatever ends up being decided. Ultimately, I just want to see this bug fixed. I'll hold off on updating this further until there's more feedback.

I do appreciate the links and all of the feedback, by the way. I wasn't aware that those changes were being made to the coding conventions.